content/en/service_management/workflows/connections.md

```diff
@@ -105,6 +106,55 @@ To learn how to restrict connection use, see [Access and Authentication][4].
 
 To connect to an arbitrary service, use the HTTP connection type. For authentication options and setup instructions, see [HTTP action][10].
 
+## Connection groups
+
+You can create groups of connections so that your workflows can authenticate into the correct account or accounts based on the given inputs. Connections can be grouped together only if they share the same integration (for example, you cannot group GCP and AWS connections within the same group). 
+
+You define the members of a connection group using a connection's _Identifier Tags_. For example, you can create a connection group consisting of AWS accounts that have the `env` tag.
```

Comment:
let's add `account_id: 123456789` as another example for AWS

Reply:
Before your example below, let's add: Each connection in the group must have a set of unique tags, so that a workflow can dynamically select the correct correction at runtime.

Reply:
I actually replaced the existing example with account_id instead.